### ISSUE TITLE
XML body consumed by skipper

### ISSUE THREAD

**Issue by gavinlee2014** [balderdashy/skipper#129]
If the http body is XML format,skipper will not handle it but still consume it.so if skipper won't deal with the XML body,please pass them through so that other middleware can handle it

**Comment by mikermcneil** [comment]
Hey @gavinlee2014, could you provide some more information about the request being sent?  In particular, seeing the content type header would be helpful.  Thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @gavinlee2014.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sgress454** [comment]
@gavinlee2014 Just posted an example of parsing XML using a custom body parser config in [Sails101](https://github.com/sails101/use-xml-body-parser).  Having Skipper ignore requests it can't handle and just pass them along is not a stable solution because we have no guarantee that there _is_ another body parser in the middleware stack--better to fail than to let actual app code run with jacked up bodies in the request.
